analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/PubliclyWritableDirectoriesBase.cs

```diff
@@ -71,6 +71,9 @@ protected PubliclyWritableDirectoriesBase(IAnalyzerConfiguration configuration)
 
         protected override void Initialize(SonarAnalysisContext context)
         {
+            var kinds = Language.SyntaxKind.StringLiteralExpressions.ToList();
+            kinds.Add(Language.SyntaxKind.InterpolatedStringExpression);
```

Comment:
Starting with .NET 4.7.1 there is [Enumerable.Append<TSource>(IEnumerable<TSource>, TSource) Method (System.Linq)](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.append?view=netframework-4.8). Can you use that instead or do we target lower versions of the framework?
```suggestion
            var kinds = Language.SyntaxKind.StringLiteralExpressions.Append(Language.SyntaxKind.InterpolatedStringExpression).ToArray();
```

Reply:
I see we are targetting `net46;netstandard2.0`.

Reply:
That's unfortunate.